ee/allowedcmd/run_disclaimed_darwin.go

```diff
@@ -0,0 +1,174 @@
+//go:build darwin
+// +build darwin
+
+package allowedcmd
```

Comment:
This is an interesting place to hang this. I don't know if it's wrong, but it'd definitely breaking my model of the world. 😆 

I'd probably ee/disclaim or something

Reply:
yeah this did end up less coupled than I had thought, I can break it out